src/sp/it/pl/util/system/Environment.kt

```diff
@@ -143,8 +144,8 @@ fun URI.browse() {
 
 /**
  * Edit the file, in order:
- * * if is a directory, [open] is called
- * * if is a file, it will be edited in default associated editor program
+ * - directory -> [open] is called
+ * - file -> open in associated editor
```

Comment:
open in system associated editor?

Also it could be good idea to add clause for when edit fails due to no associated editor open is called. This is because unlike 'open', 'edit' is an optional keyword  on windows) and hence the concept of 'edit' sort of doesnt exist unless user adds associates an editor for the extension. Because of this invoking edit can never assume that it will do anything. Perhaps it is for this reason that windows  will not even open an association window when no editor is associated. I'm not sure how Linux works, but this explains my reasoning why edit has a fallback on open.

`On some platforms the operation may be unsupported. In that case this method is a no-op.`

is therefore incorrect. No-op makes no sense.